modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectRelationshipLocalServiceTest.java

```diff
@@ -408,6 +421,37 @@ private static ObjectDefinition _addSystemObjectDefinition(
 		return systemObjectDefinition;
 	}
 
+	private ObjectRelationship _addObjectRelationshipSystemObjectDefinition()
+		throws Exception {
+
+		String objectRelationshipName = StringUtil.randomId();
+
+		_objectRelationshipLocalService.addObjectRelationship(
+			TestPropsValues.getUserId(),
+			_objectDefinition1.getObjectDefinitionId(),
+			_objectDefinition2.getObjectDefinitionId(), 0,
+			ObjectRelationshipConstants.DELETION_TYPE_PREVENT,
+			LocalizedMapUtil.getLocalizedMap(RandomTestUtil.randomString()),
+			objectRelationshipName,
+			ObjectRelationshipConstants.TYPE_ONE_TO_MANY);
+
+		ObjectField objectField3 = _objectFieldLocalService.getObjectField(
```

Comment:
@gabrielwas we don't have conflicts here, so we can rename to objectField

Reply:
https://github.com/liferay-objects/liferay-portal/pull/2418/commits/df40f2c3dcd095447c077e0fdab4385957f49c66